spk/beets/src/installer.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+
+# Package
+PACKAGE="beets"
+DNAME="beets"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+INSTALL_BIN_DIR="/usr/local/${PACKAGE}/env/bin"
+PYTHON_DIR="/usr/local/python"
+PATH="${INSTALL_DIR}/bin:${INSTALL_DIR}/env/bin:${PYTHON_DIR}/bin:${PATH}"
+VIRTUALENV="${PYTHON_DIR}/bin/virtualenv"
+TMP_DIR="${SYNOPKG_PKGDEST}/../../@tmp"
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+
+    # Create a Python virtualenv
+    ${VIRTUALENV} --system-site-packages ${INSTALL_DIR}/env > ${INSTALL_DIR}/install-details.log
+
+    # Install the wheels
+    ${INSTALL_DIR}/env/bin/pip install --no-deps --no-index -U --force-reinstall -f ${INSTALL_DIR}/share/wheelhouse ${INSTALL_DIR}/share/wheelhouse/*.whl > ${INSTALL_DIR}/install-details.log 2>&1
+
+    # Log installation details
```

Comment:
To be fair, this looks kind of messy to me. Let's leave it for now, so you can gather feedback for the time the package is in beta, but let's remove it (or a cleanup) when the beta tag is removed. Applies to this section, redirect on L26, redirect on L29.
Agree?